data/virt_autotest/tw_ipxe/menu.ipxe

```diff
@@ -1,150 +0,0 @@
-# This is the ipxe menu for installing TW on O3
```

Comment:
@Julie-CAO Is it fine to remove this menu.ipxe file?

Reply:
Yes, Wayne asked me offline. In the past, menu.ipxe on O3 was maintained by me, so I added it in github for tracking purpose, but now it is maintained by tools team, so the file is not required any more.